### PR TITLE
Open source cpe_onepassword cookbook

### PR DESCRIPTION
**Pull request by nstrauss**

### CODE REVIEW

File: chef/cookbooks/cpe_onepassword/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+cpe_onepassword Cookbook
+========================
+Manage 1Password's JSON config using the CPE attributes API pattern on macOS and Windows.
```
Comment (williamtheaker) [inline chef/cookbooks/cpe_onepassword/README.md:3]:
```suggestion
Manage 1Password 8's JSON config using the CPE attributes API pattern on macOS and Windows.
```

Comment (williamtheaker) [reply chef/cookbooks/cpe_onepassword/README.md:3]: I might be wrong, but I thought settings.json didn't exist on 7 and older.

**Review by williamtheaker — CHANGES_REQUESTED** [review]

File: chef/cookbooks/cpe_onepassword/metadata.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name "cpe_onepassword"
+maintainer "Gusto, Inc."
+maintainer_email "noreply@gusto.com"
+license 'Apache'
```
Comment (williamtheaker) [inline chef/cookbooks/cpe_onepassword/metadata.rb:4]:
```suggestion
license "Apache-2.0"
```

File: chef/cookbooks/cpe_onepassword/README.md (hunk 1 above)
Comment (nstrauss) [reply chef/cookbooks/cpe_onepassword/README.md]: https://github.com/Gusto/it-cpe-opensource/pull/77/commits/bb9a53a6a023fac87c4600691bfffd019f35a728

**Review by williamtheaker — APPROVED** [review]
Great! Congrats on getting this released :)

**Comment by nstrauss** [comment]
> Great! Congrats on getting this released :)

Thank you. 😄 Committed Windows fixes we worked on yesterday. Please re-approve.

**Review by williamtheaker — APPROVED** [review]

**Pull request merged**